src/components/test-utils/mocks/MockProviders.tsx

```
@@ -0,0 +1,31 @@
+
+export const MockI18nextProvider = ({
+  children,
+  i18n,
+}: {
+  children: ReactNode;
+  i18n: i18n;
+}) => (
+  <div data-testid="i18next-provider" data-i18n={i18n ? 'provided' : 'missing'}>
+    {children}
+  </div>
+);
```

ESLint

[error] Declare only one React component per file

(react/no-multi-comp)